### PR TITLE
Rhmap 13690 - update fh-db

### PR DESCRIPTION
**Pull request by rachael-oregan**
## Motivation
Update fh -db changes - https://github.com/feedhenry/fh-db/pull/29

## Jira 
https://issues.jboss.org/browse/RHMAP-13690

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "debug": "2.6.3",
     "eyes": "0.1.8",
     "fh-component-metrics": "2.7.0",
-    "fh-db": "2.0.0",
+    "fh-db": "git://github.com/rachael-oregan/fh-db.git#RHMAP-13690",
```
Comment (wtrocki) [inline package.json]: You can do prerelease for this purpose. Something like `2.0.1-pre.1`

**Comment by wtrocki** [comment]
No shrinkwrap

**Comment by wtrocki** [comment]
@rachael-oregan I think we will need now to hardcode some versions along with shrinkwrap. 
`npm update` command will do that for you automatically.

**Comment by rachael-oregan** [comment]
@wtrocki thanks, I'm just trying some things locally at the moment, will get back to fixing this tomorrow hopefully

**Comment by rachael-oregan** [comment]
Retest this please

**Pull request merged**